### ISSUE TITLE
Clean up Require/Import setup

### ISSUE THREAD

**Issue by dhershman1** [dhershman1/octoris#10]
This is a issue ticket to talk about cleaning up the way importing/requiring of octoris works.

Currently the idea was:
```js
const { send } = require('octoris/response')
const { GET } = require('octoris/methods')
const { route, fixed, routeReducer } = require('octoris/router')
```

But my fear is that this may become to cumbersome to fully use, I also feel like I might be making it a bit to "micro" when it comes to separation of concerns.

So a new proposed idea of handling this might be just making everything a module object apart of octoris so requiring them would be super easy:

```js
const { router, methods, response } = require('octoris')
```

There is also the theory that combining `methods` and `response` into `router` might be a solid idea but I am not 100% sold on this just yet.

I like being able to do
```js
const { router, methods, response } = require('octoris')

methods.GET
response.send
```

Vs having to do

```js
const { router } = require('octoris')

router.methods.GET
router.response.send
```

This is all subject to change and is mainly theory and ideas for now. More will come in the future.

Thoughts? Concerns? Comments?

**Comment by dhershman1** [comment]
```js
const { listen, router, methods, response } = require('../lib')
const { send } = response
const { route, routeReducer, fixed } = router
const { GET, POST } = methods
```

To me this is the same as just requiring the sub modules from octoris... Though you don't need to do this it is easier to read when building out your stuff.

```js
const home = router.route([router.fixed('home')], [methods.GET(getHandlerThing)])
```

Vs

```js
const home = route([fixed('home')], [GET(getHandlerThing)])
```

**Comment by dhershman1** [comment]
The new setup was implemented for the 0.1.0 release of the framework
